src/core/core.spec.js

```diff
@@ -53,6 +53,20 @@ describe("Core module", () => {
         core.selectOs.mockRestore();
       });
     });
+    it("should prepare and read config at restart", () => {
+      jest.spyOn(core, "readConfigFile").mockResolvedValueOnce();
+      jest.spyOn(core.plugins, "init").mockResolvedValueOnce();
+      jest.spyOn(core, "selectOs").mockResolvedValueOnce();
+      return core.prepare("a", true).then(() => {
+        expect(core.readConfigFile).toHaveBeenCalledWith("a");
+        expect(core.readConfigFile).toHaveBeenCalledTimes(1);
+        core.readConfigFile.mockRestore();
```

Comment:
You should restore the mocks in a `.finally` to make sure you never influence other test cases

Reply:
Could you provide me a bit more guidance here please? 

Is this different from the [reset before each of the tests](https://github.com/ubports/ubports-installer/blob/0eb344fc615ca4be1ec2be4339b30ded6508fd16/src/core/core.spec.js#L38)?

Reply:
should not be needed, but jest can be weird sometimes

Reply:
yep

Reply:
Just to clarify, I'm happy to make changes here, but I'd still need some guidance on what's needed.